packit_service/worker/checker/bodhi.py

```diff
@@ -65,27 +80,59 @@ class IsKojiBuildCompleteAndBranchConfiguredCheckService(
     ...
 
 
-class HasAuthorWriteAccess(ActorChecker, ConfigFromEventMixin):
+class HasIssueCommenterRetriggeringPermissions(ActorChecker, ConfigFromEventMixin):
+    """To be able to retrigger a Bodhi update the issue commenter should
+    have write permission on the project.
+    """
+
     def _pre_check(self) -> bool:
-        if not self.project.has_write_access(user=self.actor):
-            logger.info(
-                f"Re-triggering Bodhi update via dist-git comment in PR#{self.data.pr_id}"
-                f" and project {self.project.repo} is not allowed for the user: {self.actor}."
+        has_write_access = self.project.has_write_access(user=self.actor)
+        if self.data.event_type in (IssueCommentEvent.__name__,):
+            logger.debug(
+                f"Re-triggering Bodhi update through comment in "
+                f"repo {self.project.repo} and issue {self.data.issue_id} "
+                f"by {self.actor}."
             )
-            return False
+            if not has_write_access:
+                logger.warning(
+                    f"Re-triggering Bodhi update through comment in "
+                    f"repo {self.project.repo} and issue {self.data.issue_id} "
+                    f"is not allowed for the user {self.actor} "
+                    f"which has not write permissions on the project."
+                )
+                return False
```

Comment:
we should comment in the issue when this happens

Reply:
Ok! I can also add the feedback for the other checks in the same way and we can resolve the #1747